dbms/src/Storages/Page/V3/Universal/UniversalPageStorage.cpp

```diff
@@ -472,16 +462,17 @@ PS::V3::CPDataWriteStats UniversalPageStorage::dumpIncrementalCheckpoint(const U
     }
     // get the remote file ids that need to be compacted
     auto write_stats = writer->writeEditsAndApplyCheckpointInfo(edit_from_mem, file_ids_to_compact);
-    writer->writeSuffix();
+    auto data_file_paths = writer->writeSuffix();
     writer.reset();
+    auto dump_data_seconds = sw.elapsedSeconds() - dump_snapshot_seconds;
```

Comment:
can use `Stopwatch::elapsedSecondsFromLastTime` instead of calculating the diff manually